examples/temporal/main.js

```diff
@@ -0,0 +1,592 @@
+( function () {
```

Comment:
Ok, 

Note Explaining a problem here: 
I started Code with copy/paste from the sample the most like this one. (postprocess). 
Productivity side, adding new sample from scratch is very time consuming.

Using Example would give gains, but would works only if we're sharing code between example, as in inherit from a example js and then convert all Example once and for all.

Otherwise adding a new sample is taking code parts from 2 or 3 different sample, each with its particular code, and in many case it's copy/paste and hard to test in different conditions.

Here, for example, if you want to test supersample on different condition: with a cubemap, with animation, with another RTT. 
If all those code parts where accessible when you inherit example, you'd gain massive power for debug and sample app coding speed.
Otherwise  it's very time consuming.